src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlQueueClient.cs

```diff
@@ -57,7 +57,7 @@ public async Task CancelJobByIdAsync(byte queueType, long jobId, CancellationTok
 
         public virtual async Task CompleteJobAsync(JobInfo jobInfo, bool requestCancellationOnFailure, CancellationToken cancellationToken)
         {
-            using var sqlCommand = new SqlCommand() { CommandText = "dbo.PutJobStatus", CommandType = CommandType.StoredProcedure };
+            using var sqlCommand = new SqlCommand() { CommandText = "dbo.PutJobStatus", CommandType = CommandType.StoredProcedure, CommandTimeout = 300 };
```

Comment:
Wouuld be nice to set this as config in some class - then we could override at runtime if needed.